account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TEKManager.java

```diff
@@ -0,0 +1,167 @@
+package com.alchemy.aa.core;
+
+import com.alchemy.aa.core.exceptions.NoTEKException;
+import com.google.crypto.tink.InsecureSecretKeyAccess;
+import com.google.crypto.tink.KeyTemplate;
+import com.google.crypto.tink.KeysetHandle;
+import com.google.crypto.tink.TinkJsonProtoKeysetFormat;
+import com.google.crypto.tink.hybrid.HpkeParameters;
+import com.google.crypto.tink.hybrid.HpkePrivateKey;
+import com.google.crypto.tink.hybrid.HpkePublicKey;
+import com.google.crypto.tink.CleartextKeysetHandle;
+import com.google.crypto.tink.hybrid.internal.HpkeContext;
+import com.google.crypto.tink.hybrid.internal.HpkeKemKeyFactory;
+import com.google.crypto.tink.hybrid.internal.HpkePrimitiveFactory;
+
+import com.google.crypto.tink.proto.KeyData;
+import com.google.crypto.tink.proto.Keyset;
+import com.google.crypto.tink.util.Bytes;
+import com.google.crypto.tink.util.SecretBytes;
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.security.GeneralSecurityException;
+
+public class TEKManager {
+  private char [] serializedKeyset;
+
+  private static HpkeParameters _hpkeParams;
+
+  private static HpkeParameters getHpkeParams() throws GeneralSecurityException {
```

Comment:
sorry... this is super nit picky, but could we move `private` definitions to the bottom of the class definition? makes it a bit easier to quickly grok what a classes functionality is